modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestResponse.siddhi

```diff
@@ -0,0 +1,100 @@
+@App:name("PublishHttpOAuthRequestResponse")
+@App:description("sending an HTTP event via an OAuth protected endpoint and catching its response.")
+/*
+Purpose:
+	This application demonstrates how to configure WSO2 Stream Processor to sending an HTTP event via an OAuth
+	 protected endpoint and catching its response.
+Prerequisites:
+     1) Replace the Sink configuration values for following options.
+            - publisher.url : publisher URL (Ex:- 'https://localhost:8005/abc')
+            - consumer.key  : consumer key for the http request (Ex:- 'abcdef')
+            - consumer.secret: consumer secret for the http request (Ex:- 'abcdef')
+            - token.url     : URL of the token end point (Ex:-'https://localhost:8005/token')
+            - method        : method type (Eg:- POST)
+
+            optional (You can fill this if it is different from default values )
+            - header (Authorization header)  : access token for the http request (Ex:- 'abcdef') <= by default
+            it automatically get the access token using the password/client-credential/refresh grant. Here the grant
+            type depends on the user input.
+            - https.truststore.file : API trust store path (Ex:- '/user/../../../client-truststore.jks') <= by default
+             it get from product pack (Ex:- ${carbon.home}/resources/security/client-truststore.jks)
+            - https.truststore.password :  API trust store password (wso2carbon) <= by default it set as wso2carbon
+
+     2) Save this sample. If there is no syntax error, the following message is shown on the console:
+             * Siddhi App PublishHttpOAuthRequestResponse successfully deployed.
+
+Executing the Sample:
+  	1) Start the Siddhi application by clicking on 'Run'.
+  	2) If the Siddhi application starts successfully, the following messages are shown on the console:
+              * PublishHttpOAuthRequestResponse.siddhi - Started Successfully!
+              * 'Http' sink at 'LowProductionAlertStream' stream successfully connected to 'https://localhost:8005/abc'.
+
+  Testing the Sample:
+   Send events through one or more of the following methods:
+      * Send events with http server through the event simulator:
+          a) Open the event simulator by clicking on the second icon or pressing Ctrl+Shift+I.
+      	b) In the Single Simulation tab of the panel, specify the values as follows:
+                  * Siddhi App Name  : PublishHttpOAuthRequestResponse
+                  * Stream Name     : SweetProductionStream
```

Comment:
SweetProductionStream is not defined in the Siddhi app.

Reply:
SweetProductionStream added